staging/src/k8s.io/legacy-cloud-providers/azure/azure_wrap.go

```diff
@@ -7,7 +7,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
revert this change.

Reply:
sure, not sure why that changed, but will remove it